modules/bibauthority/lib/bibauthority_people_mapper.py

```diff
@@ -0,0 +1,315 @@
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+""""BibAuthority CERN people collection mapper.
+
+Map CERN LDAP records to MARC 21 authority records, create and
+write XML objects based on the MARC 21 XML Schema (MARCXML).
+"""
+
+from collections import OrderedDict
+from datetime import date
+from os import makedirs
+from os.path import dirname, exists, splitext
+
+from invenio.bibauthority_people_config import (
+    CFG_BIBAUTHORITY_AUTHOR_CERN, CFG_BIBAUTHORITY_AUTHOR_INSPIRE)
+from invenio.bibauthority_people_utils import get_inspire_id
+
+from lxml import etree
+
+
+class MapperError(Exception):
+
+    """Base class for exceptions in this module."""
+
+    pass
+
+
+class Mapper:
```

Comment:
Maybe is better to use new style classes https://docs.python.org/2/reference/datamodel.html#new-style-and-classic-classes